modules/installing-cluster-compare.adoc

```diff
@@ -0,0 +1,67 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="installing-cluster-compare_{context}"]
+= Installing the cluster-compare plugin
+
+Install the `cluster-compare` plugin to compare a reference configuration with a cluster configuration from a live cluster or `must-gather` data.
+
+.Prerequisites
+
+. You have installed the OpenShift CLI (`oc`).
+
+. You installed `podman`.
+
+. You have access to the Red Hat container catalog.
+
+.Procedure
+
+. Log in to the Red Hat container catalog by running the following command:
++
+[source,terminal]
+----
+$ podman login registry.access.redhat.com
+----
+
+. Create a container for the `cluster-compare` image by running the following command:
++
+[source,terminal]
+----
+$ podman create --name cca registry.redhat.io/openshift4/kube-compare-artifacts-rhel9:latest
+----
+
+. Copy the `cluster-compare` plugin to a directory that is included in your `PATH` environment variable by running the following command:
++
+[source,terminal]
+----
+$ podman cp cca:/usr/share/openshift/linux_amd64/kube-compare.rhel9 <folder/on/path>/kubectl-cluster_compare
```

Comment:
Maybe add a note that if you're running a RHEL8 system, you should replace `kube-compare.rhel9` with `kube-compare-rhel8`, otherwise the resulting binary won't work.

In fact, it may be useful to provide a list of all the binaries available in the released container, as we ship a set for different architectures as well.  We have a `rhel8` and `rhel9' binary for each of the following architectures:
- linux_arm64
- linux_amd64
- linux_ppc64le
- linux_s390x

Reply:
Updated accordingly, thanks!